### PR TITLE
feat(ui): add Public Sans font

### PR DESCRIPTION
**Pull request by Anjellyrika**
This PR adds the Medium, Bold and Black variants of the **Public Sans** font family.

Preview:
public-sans-preview-light
public-sans-preview-dark

### CODE REVIEW

File: src/routes/theme.css
Comment (BastiDood) [inline src/routes/theme.css:1]: Are we really gonna need _all_ of these fonts? Note that each `.woff2` file is an extra bunch of bytes over the network.

Comment (Anjellyrika) [reply src/routes/theme.css:1]: Maybe I should get rid of the italic styles?

Comment (BastiDood) [reply src/routes/theme.css:1]: We should get rid of the ones not being used by Skeleton's design system. Or just get rid of the ones that we are not currently using anyway. You can find out which fonts are used by checking out the `.woff2` files in the Network tab of the DevTools. As far as I can tell, fonts are lazily loaded, so if it doesn't appear in the Network tab (for all pages), it just means it's unused.

Comment (Anjellyrika) [reply src/routes/theme.css:1]: Okie, will remove all the unused ones. Only Medium and Bold are being rendered as of now.

Comment (BastiDood) [reply src/routes/theme.css:1]: Don't forget about the unused `@font-face` declarations.

Comment (Anjellyrika) [reply src/routes/theme.css:1]: Oh no 😅

**Review by BastiDood — CHANGES_REQUESTED** [review]
## Some Needed Changes

* Remove unused `@font-face` declarations.
* Include Public Sans to the `COPYING.md`.

**Review by BastiDood — APPROVED** [review]
LGTM! 🚀

**Pull request merged**